package.json

```diff
@@ -18,45 +18,52 @@
     }
   ],
   "scripts": {
```

Comment:
haha yes sorry bout that 
![image](https://cloud.githubusercontent.com/assets/2192970/13758042/4e20c8ac-e9f5-11e5-845a-1ed85e4bdf6c.png)